### DISCUSSION TITLE
V免签支付问题

### DISCUSSION THREAD

**Discussion by qccgithub** [Baiyuetribe/kamiFaka#59] [Bug反馈]
在V免签后台订单状态是已经完成的，但是发卡这边还是未支付状态，微信和支付宝都有这个问题。
看了文档设置网站地址也是设置对的，而且有时是正常的支付完成的
![)~7783Q}$2E5FRKE_@U}`72](https://user-images.githubusercontent.com/51011682/114325931-161e0380-9b65-11eb-8d68-90c6056b6c39.png)
![(Z8}$0$8E@ EDDJ3}H8~THU](https://user-images.githubusercontent.com/51011682/114325933-18805d80-9b65-11eb-9abd-9ab3db280875.png)

**Comment by qccgithub** [comment]
重启容器之后发现正常，测试3单之后又发现出现此问题

**Reply by Baiyuetribe** [reply]
> 重启容器之后发现正常，测试3单之后又发现出现此问题

还有这么奇怪的现象？可以尝试网站地址设置为ip加端口

**Reply by qccgithub** [reply]
设置为ip加端口后，能多测试2单了。。。然后又不行了

**Reply by vip352100** [reply]
> 设置为ip加端口后，能多测试2单了。。。然后又不行了
按照作者教程重新安装，v免签的时候，必须是支付金额有小数点的才都会正常。。。没有小数点的支付金额100%异常。

**Reply by evrstr** [reply]
我也一样，不过我一单都没测试成功，vpay后台显示完成，但是发卡后台显示未支付
